core/domain/config_domain.py

```diff
@@ -293,3 +293,7 @@ def get_config_property_schemas(cls):
     'is_improvements_tab_enabled', BOOL_SCHEMA,
     'Exposes the Improvements Tab for creators in the exploration editor.',
     False)
+
+ALWAYS_ASK_LEARNERS_FOR_ANSWER_DETAILS = ConfigProperty(
+    'always_ask_learners_for_answer_details', BOOL_SCHEMA,
+    'Always ask learners for answer details while playing exploration.', False)
```

Comment:
Please add to the description "(For testing only -- DO NOT USE)"

Reply:
Done

Reply:
Not done.

Reply:
Ah -- to be clear, I don't mean a comment. I mean in the actual description string that is shown on the admin dashboard. We need to make clear to admins that they should not turn this on.

Reply:
Done